Customer_Application/UserData/UserDataAdd.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NServiceBus;
+
+namespace UserData
+{
```

Comment:
Create seprate lib project as model and move all models thr. use the reference of model project in this project

Reply:
Created lib project as model and used reference in this project